src/openrct2/world/Location.hpp

```diff
@@ -419,6 +419,11 @@ struct TileCoordsXY
         return rotatedCoords;
     }
 
+    constexpr TileCoordsXY Offset(int32_t _x, int32_t _y) const
```

Comment:
Why is this needed? It already has operator+ implemented that does exactly this.

Reply:
it isn't _needed_, slightly more readable imo. I can remove it.